CODEOWNERS

```diff
@@ -1 +1,13 @@
-/src/lib/gadgets @o1-labs/crypto-eng-reviewers @mitschabaude
+# By default, review is required by o1js-admins
+* @o1-labs/o1js-admins
+
+# Changes to the following paths should be reviewed by o1js-admins, but we will tag current and future zkapps teams as well (incorrect use of codeowners)
+# We are relying on good faith to wait for admin approval on these PRs
+# Dangerous PRs that could release a new version, new consensus rules, or anything like that will necessarily touch more than `/src/lib`, so this is minimally risky
+/src/lib @o1-labs/o1js-admins @o1-labs/exploration-team @o1-labs/future-zkapps
```

Comment:
This incorrect use of codeowners is still a more secure subset of our current rules AFAICT.  Right now any o1 eng can approve o1js PRs.  This will require a review from a subset of that group.

Reviews from crypto, protocol, or node specialists may also be required from time to time, which can be requested manually.  We probably also want to ensure that a github admin can override and merge an emergency PR if needed.